### PR TITLE
commando-rune: show warning when creating runes with no restrictions

### PR DESCRIPTION
**Pull request by jb55**
Changelog-Added: Show warning when creating runes with no restrictions
Signed-off-by: William Casarin <jb55@jb55.com>

### CODE REVIEW

File: doc/lightning-commando-rune.7.md [hunk 1]
```diff
@@ -193,6 +193,9 @@ On success, an object is returned, containing:
 - **rune** (string): the resulting rune
 - **unique_id** (string): the id of this rune: this is set at creation and cannot be changed (even as restrictions are added)
 
+The following warnings may also be returned:
+- **warning_usage**: A warning shown when runes are created with powers that could drain your node
```
Comment (rustyrussell) [inline doc/lightning-commando-rune.7.md:197]: `warning_unrestricted_rune`?  Warning names need to be as explicit as possible...

File: plugins/commando.c [hunk 2]
```diff
@@ -804,6 +804,10 @@ static struct command_result *reply_with_rune(struct command *cmd,
 
 	json_add_string(js, "rune", rune_to_base64(tmpctx, rune));
 	json_add_string(js, "unique_id", rune->unique_id);
+
+	if (tal_count(rune->restrs) <= 1) {
+		json_add_string(js, "warning_usage", "WARNING: This rune has superadmin permissions. Anyone who has access to this rune could drain funds from your node. Be careful when giving this to apps that you don't trust. Consider using the restrictions parameter to only allow access to specific rpc methods.");
```
Comment (rustyrussell) [inline plugins/commando.c:809]: "This rune has no restrictions!" ... is clearer (and more precise IMHO!).

**Review by rustyrussell — APPROVED** [review]
Just wording...

File: doc/lightning-commando-rune.7.md (hunk 1 above)
Comment (jb55) [reply doc/lightning-commando-rune.7.md]: :+1: done

File: plugins/commando.c (hunk 2 above)
Comment (jb55) [reply plugins/commando.c]: :+1: done

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/5475/commits/fe067c47a5287e790b6b938d78c5dd95e8e71b88

**Pull request merged**